### DISCUSSION TITLE
How to get short project description?

### DISCUSSION THREAD

**Discussion by mbanders** [python-gitlab/python-gitlab#1990] [General]
## Description of the problem, including code/CLI snippet

I read https://python-gitlab.readthedocs.io/en/stable/gl_objects/projects.html

But I didn't find a reference to the short project "description".

The gitlab API mentions a short "description" for Projects here: https://docs.gitlab.com/ee/api/projects.html#create-project
I think that the same as using the web interface Settings -> General -> Project description.

Can we retrieve this short description using python-gitlab?

## Expected Behavior

Be able to retrieve the short project description.

## Specifications

  - python-gitlab version: 3.3.0
  - API version you are using (v3/v4): v4
  - Gitlab server version (or gitlab.com): 14.9

**Comment by JohnVillalovos** [comment]
I'm not sure off the top of my head. But one thing that can help is to do:

```
project = gl.projects.get(PROJECT_ID)
project.pprint()
```

**Reply by JohnVillalovos** [reply]
Based on the GitLab docs https://docs.gitlab.com/ee/api/projects.html#get-single-project

Looks like it should be in the `description` field, if a short description has been set. Looks like it can be missing.

```
project = gl.projects.get(PROJECT_ID)
print(project.description)
```

**Reply by nejch** [reply]
Yeah that should be it :) just a little typo there, `gl.projects.get(PROJECT_ID)`.

**Reply by JohnVillalovos** [reply]
> Yeah that should be it :) just a little typo there, `gl.projects.get(PROJECT_ID)`.

Thanks! Updated my original comment to add the correction. Whoops!

**Comment by mbanders** [comment]
Thank for the help! And good tips. I'll be able to explore more with that.
